src/GitHub.Exports/Models/DiffUtilities.cs

```diff
@@ -12,6 +12,16 @@ public static class DiffUtilities
 
         public static IEnumerable<DiffChunk> ParseFragment(string diff)
         {
+            // Turn Windows line endings into Unix line endings.
+            diff = NormalizeLineEndings(diff);
```

Comment:
I should create a `ContainsLooseCarriageReturn` regex/method before replacing all line endings.  This will be done for every file in PR. It's only required when there are actually loose carriage returns.